pkg/sql/internal.go

```diff
@@ -44,3 +45,15 @@ func (ie InternalExecutor) ExecuteStatementInTransaction(
 	p.session.leaseMgr = ie.LeaseManager
 	return p.exec(ctx, statement, qargs...)
 }
+
+// QueryRowInTransaction executes the supplied SQL statement as part of the
+// supplied transaction and returns the result. Statements are currently
+// executed as the root user.
+func (ie InternalExecutor) QueryRowInTransaction(
+	ctx context.Context, opName string, txn *client.Txn, statement string, qargs ...interface{},
+) (parser.Datums, error) {
+	p := makeInternalPlanner(opName, txn, security.RootUser, ie.LeaseManager.memMetrics)
+	defer finishInternalPlanner(p)
+	p.session.leaseMgr = ie.LeaseManager
+	return p.QueryRow(ctx, statement, qargs...)
+}
```

Comment:
I know we're trying to get rid of this module, but my half-baked patch to do so is proving to be too much work. Adding this method doesn't really make the situation worse, I don't think.